specificatie/zakelijk-gerechtigden.yaml

```diff
@@ -64,7 +64,7 @@ paths:
     get:
       operationId: GetZakelijkGerechtigden
       description: |
-                    Het zoeken van zakelijk gerechtigden van een kadastraal onroerende zaak. Het aandeel van de zakelijk gerechtigde wordt altijd geleverd in combinatie met het gezamenlijk aandeel (wanneer twee of meer personen een gezamenlijk aandeel hebben in een zakelijk recht, en ieders afzonderlijke aandeel in het gezamenlijk aandeel niet bekend is)
+                    Zoek zakelijk gerechtigden van een kadastraal onroerende zaak. Het aandeel van de zakelijk gerechtigde wordt altijd geleverd in combinatie met het gezamenlijk aandeel. Er is een gezamenlijk aandeel als twee of meer personen een gezamenlijk aandeel hebben in een zakelijk recht, en ieders afzonderlijke aandeel in het gezamenlijk aandeel niet bekend is.
```

Comment:
Elke zin in een eigen regel? Dit gebeurt soms wel en soms niet. Is wel makkelijker bij het lezen. Dan hoeft je niet heen en weer te scrollen